huntsman/observatory.py

```diff
@@ -325,26 +326,22 @@ def take_dark_fields(self,
            in the targets list.
```

Comment:
This description is not quite right here because this function does not look up the exposure times in the target list. Instead, this function expects an `exptimes` argument. Maybe a better description would be something like:

"Take n_darks dark frames for each exposure time specified, for each camera."